Most recent version of safe-eth-py isn't working with addresses

Using ape-safe 0.5.1 with safe-eth-py="4.3.0" and above or 0.6.0 with safe-eth-py="^4.5.0" isn't working.

Here is the error.

```
File "brownie/_cli/run.py", line 51, in main
    return_value, frame = run(
  File "brownie/project/scripts.py", line 110, in run
    return_value = f_locals[method_name](*args, **kwargs)
  File "./scripts/33_test_ape_safe.py", line 10, in main
    safe_tx = safe.multisend_from_receipts()
  File "ape_safe.py", line 115, in multisend_from_receipts
    data = MultiSend(self.multisend, self.ethereum_client).build_tx_data(txs)
  File "gnosis/safe/multi_send.py", line 199, in __init__
    assert fast_is_checksum_address(address), (
AssertionError: EthereumClient for url=http://127.0.0.1:8545 proxy factory address not valid
Terminating local RPC client...
```

--- a/ape_safe.py
+++ b/ape_safe.py
@@ -115,7 +115,9 @@ class ApeSafe(Safe):
             safe_nonce = self.pending_nonce()
 
         txs = [MultiSendTx(MultiSendOperation.CALL, tx.receiver, tx.value, tx.input) for tx in receipts]
-        data = MultiSend(self.ethereum_client, self.multisend).build_tx_data(txs)
+        data = MultiSend(
+            ethereum_client=self.ethereum_client, address=self.multisend
+        ).build_tx_data(txs)
         return self.build_multisig_tx(self.multisend, 0, data, SafeOperation.DELEGATE_CALL.value, safe_nonce=safe_nonce)
 
     def get_signer(self, signer: Optional[Union[LocalAccount, str]] = None) -> LocalAccount:

--- a/ape_safe.py
+++ b/ape_safe.py
@@ -115,7 +115,7 @@ class ApeSafe(Safe):
             safe_nonce = self.pending_nonce()
 
         txs = [MultiSendTx(MultiSendOperation.CALL, tx.receiver, tx.value, tx.input) for tx in receipts]
-        data = MultiSend(self.multisend, self.ethereum_client).build_tx_data(txs)
+        data = MultiSend(self.ethereum_client, self.multisend).build_tx_data(txs)
         return self.build_multisig_tx(self.multisend, 0, data, SafeOperation.DELEGATE_CALL.value, safe_nonce=safe_nonce)
 
     def get_signer(self, signer: Optional[Union[LocalAccount, str]] = None) -> LocalAccount:

